### DISCUSSION TITLE
Smartmeter s0 change value "Total_in"

### DISCUSSION THREAD

**Discussion by Kiter1988** [arendst/Tasmota#18752] [General]
Hello, 
Can someone explain to me how to change the two values of "Total_in" so that I have the same meter reading in Tasmota as in the electricity meter?

`>D
>B
=>sensor53 r
>W
;<style>form button{display:none}</style>
>M 2
+1,12,c,1,-10,Schlaf
1,1-0:1.8.0*255(@1000,Verbrauch,KWh,Total_in,3)
1,1-0:1.7.0*255(@0.01667, Leistung,W,Power_actual,1)
+2,14,c,1,-10,Wohn
2,1-0:1.8.0*255(@1000,Verbrauch,KWh,Total_in,3)
2,1-0:1.7.0*255(@0.01667, Leistung,W,Power_actual,1)
#`

**Comment by steand** [comment]
command: sensor53 cx value
x = number of counter
value = meter(KWh) * impulse per KWh
eq: 
sensor53 c1 5000 (if the meter shows 5 KWh and impulse is @1000)
